NebulaPatcher/Patches/Dynamic/UIMainMenu_Patch.cs

```diff
@@ -171,6 +171,18 @@ private static void OnJoinGameButtonClick()
             string ip = parts[0];
             int port;
 
+            //remove copy and paste mistakes and update the textbox to prevent user confusion in case of invalid ip address
+            ip = SanitizeIpAddressString(ip);
```

Comment:
I think using `String.Trim()` is probably better than a new function here

Reply:
Trim is identical to Trim(whitespace)